templates/partials/header.hbs

```diff
@@ -4,6 +4,7 @@
 
   {{#if home}}
     <meta name="description" content="A performance playground for JavaScript developers. Easily create and share test cases and run cross-browser benchmarks to find out which code snippet is most efficient.">
+    <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
```

Comment:
Nitpick, but let’s hardcode the `https://` prefix here, for the reasons mentioned here: https://github.com/konklone/cdns-to-https#background